nx/test/nx/defn_test.exs

```diff
@@ -2572,7 +2578,7 @@ defmodule Nx.DefnTest do
     deftransformp(multi_clause_bodiless_tf_private(1, y), do: y)
     deftransformp(multi_clause_bodiless_tf_private(x, _y), do: x)
 
-    defn(multi_clause_transform_bodiless3(a), do: multi_clause_bodiless_tf_private(a))
+    defn multi_clause_transform_bodiless3(a), do: multi_clause_bodiless_tf_private(a)
```

Comment:
@josevalim apparently there were lot's of defn(...)  already in the code. Should I leave them as they are right now or rollback?

Reply:
We should not change anything unrelated in this commit. :) but ideally all without parents!